docs/user_guide/datasets/acs_public_coverage.rst

```diff
@@ -0,0 +1,197 @@
+.. _acs-public-coverage:
+ACSPublicCoverage
+-------------------------
+
+Introduction
+^^^^^^^^^^^^^^^^^
+
+The ACSPublicCoverage dataset is one of five datasets created by Ding et al. [1]_ 
+as an improved alternative to the popular UCI Adult dataset [2]_.
+Briefly, the UCI Adult dataset is commonly used as a benchmark dataset when comparing
+different algorithmic fairness interventions. While UCI Adult is used to predict 
+whether an individual's annual income is above $50,000, the authors created 
+ACSPublicCoverage to predict whether an individual is covered by public health insurance.
+The authors compiled data from the American Community Survey (ACS) Public Use Microdata Sample (PUMS). 
+Note that this is a different source than the Annual Social and Economic Supplement (ASEC) 
+of the Current Population Survey (CPS) used to construct the original UCI Adult dataset.
+Ding et al. [#0]_ filtered the data such that ACSPublicCoverage only includes individuals under 65 years old 
+had an income of less than $30,000.
```

Comment:
```suggestion
Ding et al. [#0]_ filtered the data such that ACSPublicCoverage only includes individuals under 65 years old 
that had an income of less than $30,000.
```
Also, any idea why $30k? Is that relevant to who can get public coverage?

Reply:
Upon further research, I actually am not sure. It seems like the income requirements differ by state, so the $30,000 cutoff seems rather arbitrary, but I could have missed something.

Reply:
Oh that's interesting. Page 25 of their [paper](https://arxiv.org/pdf/2108.04884.pdf) just says that the filter was just that PINCP > $30,000.